tests/components/velux/test_config_flow.py

```diff
@@ -0,0 +1,112 @@
+"""Test the Velux config flow."""
+from __future__ import annotations
+
+from copy import deepcopy
+from typing import Any
+from unittest.mock import patch
+
+import pytest
+from pyvlx import PyVLXException
+import voluptuous as vol
+
+from homeassistant import data_entry_flow
+from homeassistant.components.velux import DOMAIN
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_USER
+from homeassistant.const import CONF_HOST, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+import homeassistant.helpers.config_validation as cv
+
+DUMMY_DATA: dict[str, Any] = {
+    CONF_HOST: "127.0.0.1",
+    CONF_PASSWORD: "NotAStrongPassword",
+}
+
+PYVLX_CONNECT_FUNCTION_PATH = "pyvlx.PyVLX.connect"
+PYVLX_DISCONNECT_FUNCTION_PATH = "pyvlx.PyVLX.disconnect"
+
+
+pytest.mark.usefixtures("mock_setup_entry")
+
+
+async def test_show_form(hass: HomeAssistant) -> None:
+    """Test that the setup form is served."""
+    result: dict[str, Any] = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER}
+    )
+
+    assert result["type"] == data_entry_flow.FlowResultType.FORM
+    assert result["step_id"] == "user"
+
+
+async def test_user_step_schema(hass: HomeAssistant) -> None:
+    """Test that the user step uses the correct Schema."""
+    result: dict[str, Any] = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER}
+    )
+
+    assert result["type"] == data_entry_flow.FlowResultType.FORM
```

Comment:
I recommend that FORM results also have checks for `result["step_id"]` and `result["errors"]`